src/Game.php

```diff
@@ -21,7 +31,7 @@ public function getEntityManager()
      * Returns the game's event manager.
      * @return EventManager The game's event manager.
      */
-    public function getEventManager()
+    public function events(): EventManager
     {
         return $this->eventManager;
     }
```

Comment:
I liked getEventManager and getEntityManager better. It's easier to see what you have (namely an instance of EventManager / EntitiyManager).
Das db return a EntityManager? The PDO connection? The information needed to connect to the database?